fullscreen.bs

```diff
@@ -13,6 +13,7 @@ Boilerplate: omit feedback-header, omit conformance, omit index, omit idl-index
 !Commits: <a href=https://github.com/whatwg/fullscreen/commits>GitHub whatwg/fullscreen/commits</a>
 !Commits: [SNAPSHOT-LINK]
 !Commits: <a href=https://twitter.com/fullscreenapi>@fullscreenapi</a>
+!Tests: <a href=https://github.com/w3c/web-platform-tests/tree/master/fullscreen>web-platform-tests/fullscreen</a> (<a href=https://github.com/w3c/web-platform-tests/issues?q=is%3Aopen+is%3Aissue+label%3Afullscreen>open issues</a>, <a href=https://github.com/w3c/web-platform-tests/pulls?q=is%3Aopen+is%3Apr+label%3Afullscreen>open PRs</a>)
```

Comment:
Maybe "Web Platform Tests fullscreen/" as link text? As for the second and third link, I suggest we replace those with https://github.com/w3c/web-platform-tests/labels/fullscreen and something like "ongoing work" as text.